packages/composite-checkout/src/components/credit-card-fields.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import Field from './field';
+import GridRow from './grid-row';
+import { useLocalize } from '../lib/localize';
+
+/**
+ * Styles
+ */
+
+const CreditCardFieldsWrapper = styled.div`
+	padding: 16px;
+	position: relative;
+
+	:after {
+		display: block;
+		width: calc( 100% - 6px );
+		height: 1px;
+		content: '';
+		background: ${props => props.theme.colors.gray5};
+		position: absolute;
+		top: 0;
+		left: 3px;
+	}
+`;
+
+const CreditCardField = styled( Field )`
+	margin-top: 16px;
+
+	:first-child {
+		margin-top: 0;
+	}
+`;
+const FieldRow = styled( GridRow )`
+	margin-top: 16px;
+`;
+
+const CVVImage = styled( CVV )`
+	margin-top: 21px;
+	display: block;
+	width: 100%;
+`;
+
+const LockIconGraphic = styled.svg`
+	width: 17px;
+	height: 17px;
+	display: block;
+`;
+
+/**
+ * Component
```

Comment:
I don't think we need these comments (`Component`, `Styles`). However, if you want to make it more clear, we could move the exported component functions to the top of the file instead. That way when the file is opened, the public parts are the first things visible.

Reply:
I'll take out the comments for now. Let's talk about this structure in our pairing session today. We can deal with it in a separate PR.